lib/v2/gov/maintainer.js

```diff
@@ -75,4 +75,5 @@ module.exports = {
     '/wuhan/sy/whyw': ['nczitzk'],
     '/xinyi/:path+': ['ShuiHuo'],
     '/xuzhou/hrss/:category?': ['nczitzk'],
+    '/chongqing/rsks': ['Mai19930513'],
```

Comment:
Please insert new entries in alphabetical order.
This also applies to `radar.js` and `router.js`